### PR TITLE
fix (ecosystem): support for 'load more'

### PR DESCRIPTION
**Pull request by Holybasil**

### CODE REVIEW

File: src/components/RenderIfVisible/index.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  visibleOffset = 1000,
+  root = null,
+  rootElement = "div",
+  rootElementClass = "",
+  children,
+}: Props) => {
+  const [isVisible, setIsVisible] = useState<boolean>(initialVisible)
+  const intersectionRef = useRef<HTMLDivElement>(null)
+
+  // if height of the items are not equal
```
Comment (OrestTa) [inline src/components/RenderIfVisible/index.tsx:27]: commented section still needed?

**Review by OrestTa — APPROVED** [review]
LGTM, but didn't @stateofkate prefer to default to infinite-scroll, with 'load more' only as bad-network fallback?

File: src/components/RenderIfVisible/index.tsx (hunk 1 above)
Comment (Holybasil) [reply src/components/RenderIfVisible/index.tsx:27]: yes, let's keep it until the new design comes out.

**Comment by Holybasil** [comment]
Unless I miss something, "Load More" button is default because Kate wants to track if users browse through our entire ecosystem by monitoring their use of the "Load More."

fyi: here is the [wireframe design](https://www.figma.com/file/E6mzbVdLiuMot6dLNwQV4P/MVP-Website-Inspo-%2B-Wireframes?type=design&node-id=116-2&t=Zq25gJR2C8iUsJBZ-0)

**Comment by OrestTa** [comment]
Are we tracking that here already though? Also, such analytics could be performed more precisely and without sacrificing ux by using scroll events and other user interaction during the session.

**Comment by Holybasil** [comment]
not yet. btw, I'm not quite sure if it's appropriate for us to enter into the wireframe design review process now.
What do you think? @stateofkate

**Comment by stateofkate** [comment]
I would like to utilize the "Load More" button, given that is the design that we have settled on for the new ecosystem page through working with Dylan + Mei on comms. Part of the reasoning for using the Load More button is also that there would be room to accomodate CTAs or information in the future down in the footer. Also an endless scroll could become information overload for folks who aren't necessarily trying to opt into that experience.

**Comment by stateofkate** [comment]
I thought it would be easier for Yulan to create a "load more" experience in the current website iteration given that we will also be implementing this in the mainnet iteration

**Pull request merged**